salt/metalk8s/orchestrate/bootstrap_step1.sls

```diff
@@ -0,0 +1,46 @@
+Run bootstrap prechecks:
+  salt.state:
+    - tgt: local
+    - sls: metalk8s.bootstrap.precheck
+
+Run bootstrap kube client:
+  salt.state:
+    - tgt: local
+    - sls: metalk8s.bootstrap.kubernetes_client
+    - pillar:
+        repo:
+          local_mode: True
```

Comment:
We don't need this for all steps?

Reply:
only for those that deal with installation